Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM php:8.1-apache
+
+
+# prep folders and perms
+RUN mkdir -p /var/www/html/episodes
+
+# prep env vars
+ENV \
+    ITEM_COUNT=10000 \
+    MP3_DIR=/var/www/html/episodes \
+    TITLE= \
+    AUTO_SAVE_COVER_ART=
+
+# add any dir2cast.ini options as env vars here
+# todo - making this reusable for all env vars
+RUN echo '; review options in dir2cast.ini source' > /var/www/html/dir2cast.ini
+RUN echo "ITEM_COUNT = ${ITEM_COUNT}" >> /var/www/html/dir2cast.ini
+RUN echo "MP3_DIR = ${MP3_DIR}" >> /var/www/html/dir2cast.ini
+RUN if [[ $TITLE ]]; then echo "TITLE = ${TITLE}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $MP3_URL ]]; then echo "MP3_URL = ${MP3_URL}" >> /var/www/html/dir2cast.ini; fi;
+
+RUN if [[ $IMAGE ]]; then echo "IMAGE = ${IMAGE}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $ITUNES_EXPLICIT ]]; then echo "ITUNES_EXPLICIT = ${ITUNES_EXPLICIT}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $DESCRIPTION ]]; then echo "DESCRIPTION = ${DESCRIPTION}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $ITUNES_AUTHOR ]]; then echo "ITUNES_AUTHOR = ${ITUNES_AUTHOR}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $RECURSIVE_DIRECTORY_ITERATOR ]]; then echo "RECURSIVE_DIRECTORY_ITERATOR = ${RECURSIVE_DIRECTORY_ITERATOR}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $AUTO_SAVE_COVER_ART ]]; then echo "AUTO_SAVE_COVER_ART = ${AUTO_SAVE_COVER_ART}" >> /var/www/html/dir2cast.ini; fi;
```

Comment:
so, I assume what you intended was for someone to be able to do e.g. `docker run  -d -p 8080:80 -v podcast_volume:/var/www/html/episodes -e IMAGE=... -e ITUNES_EXPLICIT=... ctkcoding/dir2cast:v0.1` etc etc. but, these `RUN` lines run at *build* time, not run time. 

There are two standard approaches to deal with this.

1. the user should edit dir2cast.ini and then mount it at run time with `-v dir2cast.ini:/var/www/html/dir2cast.ini`
2. use `envsubst` to replace placeholders in the baked in dir2cast.ini at run time, so you can do `-e THING=whatever` like i imagine you intended. (You will need to set a default for every single one i think). This way is harder but nicer for the user

Reply:
Updated to support ini creation through environment variables at container run time rather than image build time